src/cbmc/cbmc_parse_options.cpp

```diff
@@ -11,8 +11,8 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "cbmc_parse_options.h"
 
-#include <fstream>
 #include <cstdlib> // exit()
+#include <fstream>
```

Comment:
It looks like you called `clang-format` on the whole file: maybe a separate commit?

Reply:
Nah I'll undo it.